src/components/lists/ShotList.vue

```diff
@@ -9,7 +9,7 @@
         @minimize-clicked="onMinimizeColumnToggled()"
         @delete-all-clicked="onDeleteAllTasksClicked()"
         @sort-by-clicked="onSortByTaskTypeClicked()"
-        @select-column="onSelectColumn"
+        @select-column="onSelectColumn(displayedShots.flat())"
```

Comment:
Displayed shots don't represent all the shots. The data from the cache are needed for assets and shots.

Reply:
Correct, I forgot the data pagination.